### ISSUE TITLE
coin3d.org is down

### ISSUE THREAD

**Issue by Vanuan** [coin3d/quarter#47]
Remove the mention of coin3d.org from docs? Or is there a hope for recovery?

**Comment by Vanuan** [comment]
Some research.

Buying out coin3d.org costs 120 USD of broker costs and unknown price set by the owner. 21 USD after that

coin3d.io costs 60 USD per year
coin3d.dev costs 20 USD per year
coin-3d.org costs 21 USD per year


It looks like the cheapest option is the .dev domain.

WDYT?

**Comment by VolkerEnderlein** [comment]
coin3d.org has been shut down long ago but if I remember correctly mails to coin-support@coin3d.org are still redirected to the Coin3D maintainer Bastiaan Veelo. @veelo can you confirm that? That's why we still keep the email in the docs.
 
According to https://lookup.icann.org/lookup Kongsberg Digital AS is still the registered owner of coin3d.org domain. As they already donated Coin3D code to the public maybe we can contact them for a domain transfer to the coin3d organisation.

**Comment by Vanuan** [comment]
Yeah, would be awesome to use a dedicated domain instead of the https://coin3d.github.io/

**Comment by Vanuan** [comment]
The last version of the website is archived here:
https://web.archive.org/web/20120806074058/http://www.coin3d.org/

Has the logo been re-licensed too? Or should the be a separate community logo?

**Comment by VolkerEnderlein** [comment]
I do not know about the logo but the Coin [wiki page](https://github.com/coin3d/coin/wiki) has nearly all noncommercial information preserved.

**Comment by veelo** [comment]
>... if I remember correctly mails to [coin-support@coin3d.org](mailto:coin-support@coin3d.org) are still redirected to the Coin3D maintainer Bastiaan Veelo. @veelo can you confirm that? That's why we still keep the email in the docs.

I am not aware of this, if this is the case. I have never received a support request personally. I can imagine that it forwards to the [Google group](https://groups.google.com/g/coin3d-discuss) but I'm not sure.

**Comment by veelo** [comment]
coin-support@coin3d.org does no go anywhere any more.
```
Remote Server returned '550 Unknown user'
```

**Comment by Vanuan** [comment]
@veelo @VolkerEnderlein So, what's your suggestion? Should we wait until coin3d.org holding period expires, should we negotiate current owner to transfer rights, should we register new domain?
